### PR TITLE
chore: Adding swagger 4 plugin

### PR DESCRIPTION
**Pull request by rasoul-shahriari**

### CODE REVIEW

File: libs/users/src/users.model.ts [hunk 1]
```diff
@@ -1,36 +1,26 @@
...
-  @ApiProperty()
   public lastLoginAt?: Date;
-  @ApiProperty()
   public isAdmin?: boolean;
-  @ApiProperty()
   public isAnonymous?: boolean;
-  @ApiProperty()
-  public uuidRegTokenPair?: UuidRegTokenPair;
-  @ApiProperty()
+  public uuidRegTokenPair?: string;
```
Comment (mfateminia) [inline libs/users/src/users.model.ts:22]: this one is not string, it s this type

File: libs/profiles/src/profile.model.ts [hunk 2]
```diff
@@ -42,37 +43,29 @@ export class Location {
...
   @IsOptional()
   @IsString()
-  @ApiProperty()
   public city: string;
   @IsOptional()
   @IsString()
-  @ApiProperty()
   public country?: string;
-  @ValidateNested()
+  @ValidateNested({ each: true })
```
Comment (mfateminia) [inline libs/profiles/src/profile.model.ts:59]: does it need @IsArray() ? same issue that we had in ocean?

File: libs/users/src/users.model.ts (hunk 1 above)
Comment (rasoul-shahriari) [reply libs/users/src/users.model.ts:22]: Yes, I know but it was [key: string]: string and it's not allowed in swagger 4, if we use UuidRegTokenPair is we get empty Object in swagger model

File: libs/profiles/src/profile.model.ts (hunk 2 above)
Comment (rasoul-shahriari) [reply libs/profiles/src/profile.model.ts:59]: if its an array definitely

**Pull request merged**